dropzone/index.d.ts

```diff
@@ -178,7 +178,7 @@ declare class Dropzone {
 
 	on(eventName:string, callback:(...args:any[]) => void):Dropzone;
 
-	off(eventName:string):Dropzone;
+	off(eventName?:string, callback?:(...args:any[]) => void):Dropzone;
```

Comment:
Actually, that's the same thing.
Having the two overloads declares that either
* It could take 0 arguments
* Or it could take `eventName`
* Or it could take `eventName` and `callback`

Which is what we get with two optional parameters, right?
Merging anyway since `unified-signatures` will catch this if we ever lint this file.